basin3d/plugins/usgs.py

```diff
@@ -128,6 +128,7 @@ def generator_usgs_measurement_timeseries_tvp_observation(view,
         for stat in query.statistic:
             sythesized_stat = USGS_STATISTIC_MAP.get(stat)
             if not sythesized_stat:
+                synthesis_messages.append(f"USGS Daily Values service does not support statistic {stat}")
                 logger.info(f"USGS Daily Values service does not support statistic {stat}")
```

Comment:
Do we anticipate that the messages will be different between the synthesis_messages and logger?
If not, I suggest having a single message. Something like:
```
msg = f"USGS Daily Values service does not support statistic {stat}"
synthesis_message.append(msg)
logger.info(msg)
```

Reply:
@dschristianson you bring up a good point. My thought was to leave the logging messages in place for when we implement logger handling. I think this still needs to be thought through.